tests/e2e/e2e_test.go

```diff
@@ -297,8 +297,23 @@ var _ = Describe("odoe2e", func() {
 				curProj = strings.TrimSuffix(curProj, "\n")
 				// Sleep until status tags and their annotations are created
 				time.Sleep(15 * time.Second)
+
+				runCmdShouldPass("echo -e \".git\ntests/\nREADME.md\" >> " + tmpDir + "/nodejs-ex/.odoignore")
```

Comment:
As per the PR description i really don't understand what you are doing here. Also can you please update the pr description with proper outcome 

Copied from pr description
```
* Trigger `odo push` and check if the files in matching in .gitignore/.odoignore are pushed into the pod or not

* Trigger `odo push` with the `--ignore` flag and check if the files are pushed into the pod or not
```

Are you sure the command ```"echo -e \".git\ntests/\nREADME.md\" >> " + tmpDir + "/nodejs-ex/.odoignore"``` reporting the right info you are looking for

This is what i am seeing in my bash terminal
```
$ echo -e \".git\ntests/\nREADME.md\" >> nodejs-ex/.odoignore
$ cd nodejs-ex/ && cat .odoignore
".gitntests/nREADME.md"
```

NOTE: Use go library instead of terminal specific command.

Reply:
@mik-dass Please fix the test code as discussed ;)

Reply:
ok on it